### DISCUSSION TITLE
Patterns for lists and their individual entities

### DISCUSSION THREAD

**Discussion by TeaBough** [pmndrs/jotai#1539] [Q&A] [answered]
Hi ! There is still a usecase I can't figure out how to do properly.
Suppose I have an rest api that is giving me a list of cars.
I use jotai-react-query to put that list in a jotai atom.

I have a page where I want to display all the cars. Something that would look like this : 
```javascript
const [cars] = useAtom(carsDataAtom);

return cars.map(car => <Car key={car.id} car={car} />)
```

but I don't want this compontent to render everytime I make a change on a car. So my idea was to do something more like this : 
```javascript
const [carIds] = useAtom(carIdsDataAtom);

return carIds.map(carId => <Car key={car.id} carId={carId} />)
```

but then I don't know in the `Car` component how can I get the car object from the jotai state using the id of the car...

**Comment by dai-shi** [comment, marked as answer]
There are several options, but let me point a doc for now: https://jotai.org/docs/advanced-recipes/large-objects

**Reply by TeaBough** [reply]
Oh thx ! Sorry, missed that part of the doc...

**Reply by dai-shi** [reply]
No problem. It's just one approach. Feel free to keep posting insights and questions.
